array.go

```diff
@@ -746,6 +746,128 @@ func (a *Array) Equal(value interface{}) *Array {
 	return a.IsEqual(value)
 }
 
+// InList succeeds if array is listed by given [values...].
+// Before comparison, both array and value are converted to canonical form.
```

Comment:
Let's reword this a bit too:

```
Before comparison, both array and each value are converted to canonical form.
```

Reply:
Addressed in `Array` and others as well.